test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/SqlServerSchemaUpgradeTests.cs

```diff
@@ -50,7 +50,7 @@ public async Task GivenTwoSchemaInitializationMethods_WhenCreatingTwoDatabases_B
             }
         }
 
-        [Fact]
+        [Fact(Skip = "Disable until AB80636")]
```

Comment:
The plan is to complete [AB#80636](https://microsofthealth.visualstudio.com/Health/_workitems/edit/80636) before I merge the `feature/sql-reindex` feature branch.